gamemodes/terrortown/gamemode/server/sv_karma.lua

```diff
@@ -115,6 +135,110 @@ function KARMA.IsEnabled()
 	return GetGlobalBool("ttt_karma", false)
 end
 
+---
+-- Allows to change the live KARMA from anywhere
+-- Use this function only if you want to influence KARMA per event or similar
+-- @note Reason is a string and will be displayed in the roundendscreen as tooltip, so use language localization or describe it properly
+-- @param Player ply
+-- @param number amount
+-- @param string reason 
+-- @realm server
+function KARMA.DoKarmaChange(ply, amount, reason)
+	reason = isstring(reason) and reason or KARMA.Reason[KARMA_UNKNOWN]
+	ply:SetLiveKarma(math.max(math.min(ply:GetLiveKarma() + amount, config.max:GetFloat()), 0))
+	KARMA.SaveKarmaChange(ply, amount, reason)
+end
+
+---
+-- Saves changes to the live KARMA
+-- @param Player ply
+-- @param number amount
+-- @param string reason 
+-- @realm server
+-- @internal
+function KARMA.SaveKarmaChange(ply, amount, reason)
+	amount = math.Round(amount,1)
```

Comment:
```suggestion
	amount = math.Round(amount, 1)
```

Reply:
`math` can be localized

Reply:
Should already be localized. Or do you mean math.Round?